### PR TITLE
Added support for single margin and border for VerticalTile layout

### PR DESCRIPTION
**Pull request by matthewsia98**
added single_margin and single_border_width keys for VerticalTile layout similar to MonadTall

### CODE REVIEW

**Comment by m-col** [comment]
Nice thanks! It looks like within that method we're checking `n > 1` a few times - perhaps that could simplified to just one if statement? Also it might be good at add the new option to the changelog so that others can see the change :)

**Comment by elParaguayo** [comment]
Are we being consistent with names across layouts?

**Comment by matthewsia98** [comment]
Names were taken from the MonadTall layout

image

**Comment by elParaguayo** [comment]
Thanks. Could remember if we used `single_margin` or `margin_on_single`.

**Comment by elParaguayo** [comment]
We should add a test for this.

**Comment by matthewsia98** [comment]
> Nice thanks! It looks like within that method we're checking `n > 1` a few times - perhaps that could simplified to just one if statement? Also it might be good at add the new option to the changelog so that others can see the change :)

Done and done.

**Comment by matthewsia98** [comment]
> Thanks. Could remember if we used `single_margin` or `margin_on_single`.

That's probably because the Columns layout uses margin_on_single.

image

File: libqtile/layout/verticaltile.py [hunk 1]
```diff
@@ -118,21 +124,21 @@ def configure(self, window, screen_rect):
             index = self.clients.index(window)
 
             # border
-            if n > 1:
-                border_width = self.border_width
-            else:
-                border_width = 0
+            border_width = self.border_width if n > 1 else self.single_border_width
```
Comment (m-col) [inline libqtile/layout/verticaltile.py:127]: Might also be worth collapsing these if statements, border, margin, width and height

Comment (matthewsia98) [reply libqtile/layout/verticaltile.py:127]: done

Comment (matthewsia98) [reply libqtile/layout/verticaltile.py:127]: If you're talking about the ```__init__``` function, I was trying to keep it same as MonadTall

image

**Pull request closed without merging**